<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(123, 130, 255), stop:1 rgb(167,187,236));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0 0, 255);</string>
    </property>
    <property name="text">
     <string>Add Jewlery</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(123, 130, 255), stop:1 rgb(167,187,236));</string>
    </property>
    <widget class="QTextEdit" name="id_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="type_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="price_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="weight_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="nr_jewels_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Id:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Price:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Weight:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Nr of jewels:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AddButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(123, 130, 255), stop:1 rgb(167,187,236));</string>
    </property>
    <widget class="QPushButton" name="Del_last">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Delete last</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Delete by Id:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="del_id_txt">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del_id">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0 0, 255);</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(123, 130, 255), stop:1 rgb(167,187,236));</string>
    </property>
    <widget class="QComboBox" name="filter_box">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Price</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Weight</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nr of Jewels</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Filter by:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="filter_crit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="filter_crit_button">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:color-rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>579</x>
      <y>199</y>
      <width>351</width>
      <height>271</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(123, 130, 255), stop:1 rgb(167,187,236));
border-color: rgb(0, 0, 0);
</string>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Index:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="index_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="id_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="weight_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="nr_jewels_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>150</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="price_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="type_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updat_button">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Nr of jewels:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Id:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Weight:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color-rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Price:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0 0, 255);</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undo_button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>500</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Redo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0 0, 255);
font: 700 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-style:italic;&quot;&gt;Shiny Jewels&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>110</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0 0, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-style:italic; color:#6c6c6c;&quot;&gt;Quality Jewels and Jewlery&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0 0, 255);</string>
    </property>
    <property name="text">
     <string>(  Type, id - string; </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0 0, 255);</string>
    </property>
    <property name="text">
     <string>   Price - float;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>580</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0 0, 255);</string>
    </property>
    <property name="text">
     <string>Weight, number of jewels - double  )</string>
    </property>
   </widget>
   <widget class="QPushButton" name="print_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Print all</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="list">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>340</y>
      <width>521</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>label</zorder>
   <zorder>widget_2</zorder>
   <zorder>widget_3</zorder>
   <zorder>widget_4</zorder>
   <zorder>label_11</zorder>
   <zorder>undo_button</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>label_13</zorder>
   <zorder>label_14</zorder>
   <zorder>label_7</zorder>
   <zorder>label_9</zorder>
   <zorder>label_15</zorder>
   <zorder>label_16</zorder>
   <zorder>label_17</zorder>
   <zorder>print_button</zorder>
   <zorder>list</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1037</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
